### DISCUSSION TITLE
Merge all the official Tailwind plugins into Tailwind

### DISCUSSION THREAD

**Discussion by lloydjatkinson** [tailwindlabs/tailwindcss#10298] [Ideas]
Whenever I make a new project with Tailwind I am always forgetting to install `typography` and `forms`.

I find this to be a bit annoying, as lets face it 99% of Tailwind designs are going to need at least forms, with many needing typography too. As they are still utility classes they won't be added to the sites dist if they aren't used, so I think that isn't a concern.

For TailwindUI users in particular where virtually every design uses `forms`, wouldn't it make sense to make it officially part of Tailwind proper?

**Comment by 52461** [comment]
Hello,
       I'm in a hurry to learn Tailwind and at least like it. I've started to work on a component library in my job, but I wasn't the one choosing the technologies and Tailwind was picked. [MyGeorgiaSouthern](https://www.mygeorgiasouthern.net/
)

**Reply by lloydjatkinson** [reply]
What the hell does that have to do with the issue at hand? Please delete this spam.

**Reply by cannap** [reply]
spam later he or the bot will edit the post with a link
